src/relentless/simulate/simulate.py

```diff
@@ -98,12 +98,12 @@ def _cantor_pairing(self, connection, neighbor):
             connection.
         """
 
-        pi_connection = (connection[:, 0] + connection[:, 1]) * (
-            connection[:, 0] + connection[:, 1] + 1
-        ) / 2 + connection[:, 1]
-        pi_neighbor = (neighbor[:, 0] + neighbor[:, 1]) * (
-            neighbor[:, 0] + neighbor[:, 1] + 1
-        ) / 2 + neighbor[:, 1]
+        pi_connection = (connection[:, 0] + connection[:, -1]) * (
+            connection[:, 0] + connection[:, -1] + 1
+        ) / 2 + connection[:, -1]
+        pi_neighbor = (neighbor[:, 0] + neighbor[:, -1]) * (
+            neighbor[:, 0] + neighbor[:, -1] + 1
+        ) / 2 + neighbor[:, -1]
```

Comment:
Since the cantor pairing is really only meant to work on 2-tuples, I would prefer that you fix the code that calls this method to only use the first and last index, rather than change the cantor pairing function. I think when you call it, you could do like:
```
_cantor_pairing(dihedrals[:, (0, -1)], neighbors)
```
to pull the first and last columns of every row.